pallets/handles/src/tests/validator_tests.rs

```diff
@@ -0,0 +1,78 @@
+use crate::utils::validator::HandleValidator;
+
+#[test]
+fn test_is_reserved_handle_happy_path() {
+	let reserved_handles: Vec<&str> = vec!["admin", "everyone", "all"];
```

Comment:
This test needs to use non-Latin character code examples, including Latin-Extended and right-to-left like Arabic and Hebrew.

Reply:
We have different examples in `handle_creation_tests.rs` file